app/services/remark_generators/residual_balance_checker.rb

```diff
@@ -21,7 +22,7 @@ def residual_balance?
     end
 
     def capital_exceeds_lower_threshold?
-      assessment.capital_summary.assessed_capital > lower_capital_threshold
+      @assessed_capital.to_f > lower_capital_threshold
```

Comment:
do we need the to_f call here? A lot of the to_f calls were due to the JSON streaming of BigDecimal with default precision

Reply:
In this case it's because `@assessed_capital` is sometimes nil (if we haven't done a capital assessment, for example). But there are more explicit ways of handling this possibility, so I will change this.